docs/sources/developers/plugins/_index.md

```diff
@@ -2,81 +2,52 @@
 aliases:
   - ../plugins/developing/
 description: Resources for creating Grafana plugins
-title: Build a plugin
+title: Welcome to the Grafana plugin developer's guide
 weight: 200
 ---
 
-# Build a plugin
+# Grafana plugin developer's guide
 
-For more information on the types of plugins you can build, refer to the [Plugin Overview]({{< relref "../../administration/plugin-management/" >}}).
+Grafana is an open source software project for creating, exploring, and sharing data through dashboards. Plugins extend Grafana's built-in functionality so that users can accomplish specialized tasks to make Grafana even more useful for them. 
 
-## Get started
+Many plugins are available in the [Catalog](https://grafana.com/grafana/plugins/) and others are built by developers for private use or to contribute them to the open source community. 
```

Comment:
To me this implies that those in the Catalog are not built by developers / contributed to the open source community. 

Suggestion (may need more wordsmithing): Developers can create plugins for private use or contribute them to the open source community, such as by publishing to the Grafana Catalog alongside hundreds of other community and commercial plugins.

Reply:
Issue is fixed in latest commit.